consensus/state.go

```diff
@@ -1470,7 +1470,22 @@ func (cs *ConsensusState) recordMetrics(height int64, block *types.Block) {
 			missingValidators++
 			missingValidatorsPower += val.VotingPower
 		}
+
+		_, privVal := cs.LastValidators.GetByAddress(cs.privValidator.GetPubKey().Address())
+		if privVal != nil && bytes.Equal(val.Address, privVal.Address) {
+			label := []string{
+				"validator_address", privVal.Address.String(),
+			}
+			cs.metrics.ValidatorPower.With(label...).Set(float64(val.VotingPower))
+			if vote != nil {
+				cs.metrics.ValidatorLastSignedHeight.With(label...).Set(float64(height))
+			} else {
+				cs.metrics.ValidatorMissedBlocks.With(label...).Add(float64(1))
+			}
+			break
```

Comment:
```suggestion
			break
```

break only works with `for` loop